libkbfs/folder_block_ops.go

```diff
@@ -959,6 +961,21 @@ func (fbo *folderBlockOps) Read(
 		lastByteInBlock := startOff + blockLen
 
 		if nextByte >= lastByteInBlock {
+			if nextBlockOff > 0 {
+				fill := nextBlockOff - startOff
```

Comment:
I _think_ this should be:

```
fill := nextBlockOff - nextByte
```

Since `startOff` refers to the start of the block that contains the next byte to be read, but we don't necessarily want to read the block from the start.